docs/root/intro/version_history.rst

```diff
@@ -10,6 +10,7 @@ Version history
 * http: added the ability to reject HTTP/1.1 requests with invalid HTTP header values, using the runtime feature `envoy.reloadable_features.strict_header_validation`.
 * tls: added verification of IP address SAN fields in certificates against configured SANs in the
   certificate validation context.
+* grpc-json: added support for :ref:`ignoring unknown query parameters<envoy_api_field_config.filter.http.transcoder.v2.GrpcJsonTranscoder.ignore_unknown_query_parameters>`.
```

Comment:
Sorry, a nit, could you arrange it in alphabetical order? Thank you.

Reply:
Sure. Do you want me to move `http` in front of `listener` too?

Reply:
Done